src/EndevorController.ts

```diff
@@ -212,7 +213,12 @@ export class EndevorController {
         this.updateIDs(connName);
       } else {
         const endevorDataProvider = new EndevorDataProvider();
-        endevorDataProvider.addSession(connName);
+        try {
+          endevorDataProvider.addSession(connName);
+        } catch (error) {
+          const errMsg = `Profile ${connName} defined in settings.json was not found. Make sure the profile is defined or remove it from settings.json. \n\n ${error.stack}`;
+          logger.warn(errMsg, error.message);
```

Comment:
Upsi, wrong order. Switched it so that user doesn't get the stack trace in the notification window.

```suggestion
          logger.warn(error.message, errMsg);
```